[Term Entry] Kotlin Math .sinh() (#3677)
Codecademy/docs: content/kotlin/concepts/math-methods/terms/sinh/sinh.md

---

> Title: 'Math.sinh()'

Reviewer: ```suggestion
Title: 'sinh()'
```

Description: 'Calculates the hyperbolic sine of a given number.'
Subjects:

>   - 'Mathematics'
>   - 'Programming'

Reviewer: ```suggestion
  - 'Code Foundations'
```

Reviewer: There should only be items from our subjects.md listed here (2 item max)

  - 'Computer Science'
Tags:
  - 'Math'
  - 'Functions'
CatalogContent:
  - 'learn-kotlin'
  - 'paths/computer-science'
---

> The **`Math.sinh()`** method in Kotlin's `Math` class is used to calculate the hyperbolic sine of a given number.

Reviewer: ```suggestion
The **`math.sinh()`** method in Kotlin's `math` class is used to calculate the hyperbolic sine of a given number.
```

## Syntax

```pseudo

> val hyperbolicSine = Math.sinh(number)

Reviewer: ```suggestion
import kotlin.math.sinh
val hyperbolicSine = sinh(number)
```

```

> - `number`: The number for which you want to calculate the hyperbolic sine.

Reviewer: ```suggestion
- `number`: The number, of type `Double` or `Float`, to be used in the calculation of the hyperbolic sine.
```

> The `Math.sinh()` method returns a value of type `Double` representing the hyperbolic sine of the input number.

Reviewer: ```suggestion
```

## Example

> This example shows how to use the `Math.sinh()` method to calculate the hyperbolic sine of a number:

Reviewer: ```suggestion
This example shows how to use the `sinh()` method to calculate the hyperbolic sine of a number:
```

> ```kotlin

Reviewer: ```suggestion
```kotlin
import kotlin.math.sinh
```

fun main() {
    val number = 2.0

>     val hyperbolicSine = Math.sinh(number)

Reviewer: ```suggestion
    val hyperbolicSine = sinh(number)
```
